lnwire/announcement_signatures_2.go

```diff
@@ -0,0 +1,97 @@
+package lnwire
+
+import (
+	"bytes"
+	"io"
+)
+
+// AnnounceSignatures2 is a direct message between two endpoints of a
+// channel and serves as an opt-in mechanism to allow the announcement of
+// a taproot channel to the rest of the network. It contains the necessary
+// signatures by the sender to construct the channel_announcement_2 message.
+type AnnounceSignatures2 struct {
+	// ChannelID is the unique description of the funding transaction.
+	// Channel id is better for users and debugging and short channel id is
+	// used for quick test on existence of the particular utxo inside the
+	// blockchain, because it contains information about block.
+	ChannelID ChannelID
+
+	// ShortChannelID is the unique description of the funding transaction.
+	// It is constructed with the most significant 3 bytes as the block
+	// height, the next 3 bytes indicating the transaction index within the
+	// block, and the least significant two bytes indicating the output
+	// index which pays to the channel.
+	ShortChannelID ShortChannelID
+
+	// PartialSignature is the combination of the partial Schnorr signature
+	// created for the node's bitcoin key with the partial signature created
+	// for the node's node ID key.
+	PartialSignature PartialSig
```

Comment:
Reloading context here, but don't we also need new nonces for the open+accept channel messages? Which are only set if the channel is to be announced.

Reply:
Also: TLV errwhere?